components/ai_chat/resources/page/stories/style.module.scss

```
@@ -8,7 +8,7 @@
   width: calc(100vw - 30px);
   height: 95vh;
   border: 1px solid var(--leo-color-divider-subtle);
-  font-family: Poppins, sans-serif;
+  font-family: sans-serif;
   font-size: 14px;
   font-weight: 400;
   line-height: 24px;
```

So for example whoever did this originally was holding it wrong - this should be something like

```suggestion
font: ${font.default.regular};
```